js/map.js

```diff
@@ -188,6 +210,25 @@ function addLayer(map, layerSource, visible = 'none'){
   })
 }
 
+function showLayer(selectedGeography, selectedCategory, selectedStatus) {
+  map.setLayoutProperty(selectedGeography + '-fills', 'visibility', 'visible')
+  map.setPaintProperty(selectedGeography + '-fills', 'fill-color', getFillColor(selectedGeography, selectedCategory, selectedStatus))
+  updateLegend(selectedGeography, selectedCategory, selectedStatus)
+}
+
+function loadParam(param_name, param_default){
```

Comment:
I'd encourage adding doc blocks to your functions - otherwise I don't think anyone else is going to be able to figure out what your intending with each one

Reply:
good call